client-registry-api/src/main/java/uk/gov/di/authentication/clientregistry/entity/ClientRegistrationRequest.java

```diff
@@ -29,11 +30,18 @@ public class ClientRegistrationRequest {
     @Required
     private List<String> contacts;
 
+    @SerializedName("public_key_source")
+    @Expose
+    private String publicKeySource = PublicKeySource.STATIC.getValue();
+
     @SerializedName("public_key")
     @Expose
-    @Required
     private String publicKey;
 
+    @SerializedName("jwks_url")
```

Comment:
```suggestion
    @SerializedName("jwks_uri")
```
See https://openid.net/specs/openid-connect-registration-1_0.html#:~:text=Section%C2%A02.1.-,jwks_uri,-OPTIONAL.%20URL%20for

Reply:
Fixed in request and response classes